docs/viewlifecycle.md

```diff
@@ -896,3 +897,21 @@ destroyed.
 
 Fired after the entire destruction process is complete. At this point, the view
 has been removed from the DOM completely.
+
+
+## Advanced Event Settings
+Marionette is able to trigger `attach`/`detach` events down the view tree along with
+triggering the `dom:refresh`/`dom:remove` events because of the view event monitor.
+This monitor starts when a view is created or shown in a region (to handle non-Marionette views).
+
+In some cases it may be a useful performance improvement to disable this functionality.
+Doing so is as easy as setting `monitorViewEvents: false` on the view class.
+
+```javascript
+const NonMonitoredView = Mn.View.extend({
+  monitorViewEvents: false
+});
+```
+
+**Important Note**: Disabling the view monitor will effectually break the monitor generated
+events for this view _and all children_ of this view. Disabling should be done carefully.
```

Comment:
By "all children" do you mean the result of `NonMonitoredView.extend()` or all child views? It's not clear from this documentation to me.